src/app/header/tabs/tabs.component.ts

```diff
@@ -78,12 +78,11 @@ export class TabsComponent implements OnDestroy {
   private _updateSelectedIfNeeded(): void {
     if (
       this._indexToSelect === undefined ||
-      isNaN(this._indexToSelect) ||
       this._currentTabs.length === 0 ||
-      (this._selectedIndex !== undefined &&
```

Comment:
if selected index is undefined, it won't match index to select due to first condition in this `if` clause